src/app/components/client/toolbar/AppPicker.tsx

```diff
@@ -0,0 +1,373 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import {
+  useMenuTriggerState,
+  useTreeState,
+  TreeState,
+  Item,
+  MenuTriggerProps,
+  OverlayTriggerState,
+  Node,
+} from "react-stately";
+import {
+  useMenuTrigger,
+  useButton,
+  useMenu,
+  DismissButton,
+  useMenuItem,
+  AriaMenuProps,
+  AriaPopoverProps,
+  usePopover,
+  Overlay,
+} from "react-aria";
+import { Key, ReactNode, useRef, RefObject } from "react";
+import Image from "next/image";
+import styles from "./AppPicker.module.scss";
+import FirefoxLogo from "./images/fx.png";
+import RelayLogo from "./images/relay-logo.svg";
+import PocketLogo from "./images/pocket.png";
+import VpnLogo from "./images/vpn.svg";
+import FxDesktopLogo from "./images/fx-logo.svg";
+import FxMobileLogo from "./images/fx-mobile.png";
+import { useL10n } from "../../../hooks/l10n";
+import { BentoIcon } from "../../server/Icons";
+import { gaEvent } from "../../../functions/client/gaEvent";
+
+const getProducts = (referringHost: string) => ({
+  relay: {
+    id: "monitor",
```

Comment:
Q: should this be `"monitor"` or `"relay"`?

Reply:
It most certainly should be `relay`: d9fa25d79